src/client.ts

```diff
@@ -386,4 +386,10 @@ export class Client {
   private storeInGlobal(): void {
     global.__APPSIGNAL__ = this
   }
+
+  tracer(): void {
+    console.error(
+      "The `appsignal.tracer()` function was called, but it has been removed in AppSignal for Node.js package version 3.x. Please read our migration guide to upgrade to this new version of our package: https://docs.appsignal.com/nodejs/3.x/migration-guide.html. It is also possible to downgrade to version 2.x, after which this code will work again."
```

Comment:
It might be good to mention that one of our other packages (`@appsignal/express` and the like) might be the one calling this function, and that they should remove them? The migration guide does say that, so it might be fine.

> It is also possible to downgrade to version 2.x, after which this code will work again.

This is true, but we may not want to point users in that direction.

Reply:
When I see error as a user I usually want two options: a way to make it go a way by following a guide, or make it go away by downgrading. I can remove the last line, but that's the two options the reader has when reading this.

Reply:
Yeah, that makes sense. 👍